ament_index_cpp/src/get_resources.cpp

```diff
@@ -46,10 +46,18 @@ get_resources(const std::string & resource_type)
     }
     dirent * entry;
     while ((entry = readdir(dir)) != NULL) {
-      // ignore folders starting with a dot
+      // ignore directories/files starting with a dot
       if (entry->d_name[0] == '.') {
         continue;
       }
+
+      // ignore directories
+      auto subdir = opendir((path + "/" + entry->d_name).c_str());
```

Comment:
ok now it's ensured that if `subdir` is `null`, it's just because it was not a directory